pkgs/development/r-modules/default.nix

```
@@ -395,6 +395,7 @@ let
     RcppZiggurat = [ pkgs.gsl ];
     reprex = [ pkgs.which ];
     rgdal = with pkgs; [ proj.dev gdal ];
+    gdalcubes = [ pkgs.pkg-config pkgs.proj pkgs.gdal pkgs.sqlite.dev pkgs.netcdf ];
```

```suggestion
    gdalcubes = with pkgs; [ pkg-config proj.dev gdal sqlite.dev netcdf ];
```
I checked, and `with` statements are used extensively in r-modules to shorten lists and make them more readable. So I think it's OK to follow this pattern, as long as we don't nest them.

---

Otherwise, the package compiled and worked.
```r
> gdalcubes::gdalcubes_version()
$VERSION_MAJOR
[1] 0

$VERSION_MINOR
[1] 3

$VERSION_PATCH
[1] 2

$BUILD_DATE
[1] "Apr 14 2023"

$BUILD_TIME
[1] "11:50:05"

$GIT_DESC
[1] "v0.3.0-45-gdf51e935"

$GIT_COMMIT
[1] "df51e9352d60d399f7186d870421fa9f523f24d2"
```